thirdparty/openapi3/schemas/UpdateThirdpartyAuthorizationRequest.yaml

```diff
@@ -0,0 +1,32 @@
+title: UpdateThirdpartyAuthorizationRequest
```

Comment:
Should this be renamed to ThirdPartyAuthorization**IDPut**Request to fall in line with other Mojaloop fspiop definitions?